third_party/vesc_uart/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+cc_library(
+    name = "vesc_uart",
+    srcs = [
+        "VescUart.cpp",
+        "WProgram.h",
+        "buffer.cpp",
+        "buffer.h",
+        "crc.cpp",
+        "crc.h",
+        "datatypes.h",
+        "local_datatypes.h",
+    ],
+    hdrs = ["VescUart.h"],
+    copts = [
```

Comment:
some of these options may not be necessary with use of  `includes`

and we'll probably want to include as a system header so warnings don't need to be suppressed by downstream targets

Reply:
Yep. I have restructured the code a bit